### PR TITLE
Dts 21835 change layout v2

### PR DESCRIPTION
**Pull request by brahmanand1**

### CODE REVIEW

File: UI/src/app/dashboard/filter/filter.component.ts [hunk 1]
```diff
@@ -25,1030 +25,1584 @@ import { GoogleAnalyticsService } from '../../services/google-analytics.service'
...
+
+   // for making the header sticky on scroll
+   @HostListener('window:scroll', [])
+   onWindowScroll() {
+     if (
+       this.router.url.indexOf('/Config/') === -1 &&
+       this.router.url !== '/dashboard/Maturity' &&
+       this.router.url !== '/dashboard/EngineeringMaturity'
+     ) {
+      console.log("inside true",this.headerFixed)
```
[the remark this answers is not in the document]
Comment (brahmanand1) [reply UI/src/app/dashboard/filter/filter.component.ts]: removed

File: UI/src/app/dashboard/filter/filter.component.spec.ts [hunk 2]
```diff
@@ -1097,5 +1097,73 @@ describe('FilterComponent', () => {
     component.showTooltip(true);
     expect(component.isTooltip).toBe(true);
    })
+
+  //  it('checking logout functionality ', (done) => {
+    //   const navigateSpy = spyOn(router, 'navigate');
+    //   component.logout();
+    //   httpMock.expectOne(baseUrl + '/api/userlogout').flush(null);
+    //   expect(navigateSpy).toHaveBeenCalledWith(['./authentication/login']);
```
[the remark this answers is not in the document]
Comment (brahmanand1) [reply UI/src/app/dashboard/filter/filter.component.spec.ts]: removed

File: UI/src/app/dashboard/dashboard.component.css [hunk 3]
```diff
@@ -16,28 +16,120 @@
...
+  -o-transition: all 0.5s ease;
+  transition: all 0.5s ease;
+}
+
+
+
+#wrapper.toggled #page-content-wrapper {
+  /* position: absolute; */
+  /* margin-right: -250px; */
+  margin-left: 11rem;
```
[the remark this answers is not in the document]
Comment (brahmanand1) [reply UI/src/app/dashboard/dashboard.component.css:129]: As discussed sidebar have fixed position so it was overlapping with rest width so given margin.

File: UI/src/app/dashboard/filter/filter.component.html [hunk 4]
```diff
@@ -1,225 +1,240 @@
...
-      <!-- Download excel Button error message start -->
-      <span class="float-right errMsz alert alert-danger" style="padding: 6px 17px;"
-         *ngIf="( enginneringMaturityErrorMessage !== '' ) && ( selectedTab === 'EngineeringMaturity' )">{{
-         enginneringMaturityErrorMessage }}</span>
-      <!-- Download excel Button error message end -->
    </div>
 </div>
-<div class="clearfix" *ngIf="selectedTab?.toLowerCase() !== 'backlog'">
 
+<div class="clearfix" *ngIf="selectedTab?.toLowerCase() !== 'backlog' && (!router.url.split('/').includes('Config') && router.url !== '/dashboard/Error' && router.url !== '/dashboard/Help')" style="margin-left: 4.8rem;">
```
[the remark this answers is not in the document]
Comment (brahmanand1) [reply UI/src/app/dashboard/filter/filter.component.html:237]: As discussed sidebar have fixed position so it was overlapping with rest width so given margin.

**Review by BhagyashreeShettigar — APPROVED** [review]

**Pull request merged**